terraform/cloud.gov.tf

```diff
@@ -1,19 +1,16 @@
-variable "cloudfoundry_elb_logging_development" {
-  default = "dualstack.development-CloudFoundry-Logging-1588361105.us-gov-west-1.elb.amazonaws.com"
-}
+locals {
+  cloud_gov_cloudfront = "d2vy872d33xc5d.cloudfront.net"
+  cloud_gov_green_elb = "dualstack.cf-green-elb-1b9dvw4o0ubi0-1450531403.us-east-1.elb.amazonaws.com."
```

Comment:
Seems like this doesn't exist anymore. Can we drop it @sharms ?

Reply:
Just to keep this pull request simpler, trying to avoid any additional cleanup. Would love to help with it as a follow-up, though!